src/saving/SaveHelper.cs

```diff
@@ -37,6 +52,12 @@ public enum SaveOrder
         FileSystem,
     }
 
+    /// <summary>
+    ///   Checks whether the last save is made within a timespan of set duration.
+    /// </summary>
+    /// <returns>True if the last save is still recent, false if otherwise or reset.</returns>
+    public static bool SaveIsRecentlyPerformed { get; private set; }
```

Comment:
Instead of having a timer, why not store the current DateTime and compare if that's more than TimeSpan.FromSeconds(x) (with that TimeSpan being stored in Constants as a static variable)?

Reply:
The last save needs to be nullable so the type should be `DateTime?`.